rust/toolchain.bzl

```diff
@@ -17,6 +27,42 @@ def _rust_toolchain_impl(ctx):
     for k, v in ctx.attr.debug_info.items():
         if not k in ctx.attr.opt_level:
             fail("Compilation mode {} is not defined in opt_level but is defined debug_info".format(k))
+    link_inputs = []
+    std_rlibs = [f for f in ctx.attr.rust_lib.files.to_list() if f.basename.endswith(".rlib")]
+    cc_toolchain, feature_configuration = find_cc_toolchain(ctx)
+    if std_rlibs:
+        link_inputs.append(cc_common.create_linker_input(
+            owner = ctx.attr.rust_lib.label,
+            libraries = depset(
+                [
+                    cc_common.create_library_to_link(
+                        # TODO(augie): wat? this needs no actions
```

Comment:
In some situations create_library_to_link will create a symlink for each shared library into one big directory so runtime linker doesn't have to look for libraries in multiple directories.